implementations/rust/ockam/ockam_command/src/project/version.rs

```diff
@@ -0,0 +1,71 @@
+use clap::Args;
+
+use ockam::Context;
+use ockam_api::cli_state::{StateDirTrait, StateItemTrait};
+use ockam_api::cloud::project::Project;
+
+use crate::node::util::{delete_embedded_node, start_embedded_node};
+use crate::project::util::refresh_projects;
+use crate::util::api::{self, CloudOpts};
+use crate::util::{node_rpc, RpcBuilder};
+use crate::{docs, CommandGlobalOpts};
+
+const LONG_ABOUT: &str = include_str!("./static/version/long_about.txt");
+const AFTER_LONG_HELP: &str = include_str!("./static/version/after_long_help.txt");
+
+//get version
+#[derive(Clone, Debug, Args)]
+#[command(
+    long_about=docs::about(LONG_ABOUT),
+    after_long_help=docs::about(AFTER_LONG_HELP)
+)]
+pub struct VersionCommand {
+    /// Name of the project.
+    #[arg(display_order = 1001)]
+    pub name: String,
+
+    #[command(flatten)]
+    pub cloud_opts: CloudOpts,
+}
+
+impl VersionCommand {
+    pub fn run(self, options: CommandGlobalOpts) {
+        node_rpc(rpc, (options, self));
+    }
+}
+
+async fn rpc(
+    mut ctx: Context,
+    (opts, cmd): (CommandGlobalOpts, VersionCommand),
+) -> miette::Result<()> {
+    run_impl(&mut ctx, opts, cmd).await
+}
+
+async fn run_impl(
+    ctx: &mut Context,
+    opts: CommandGlobalOpts,
+    cmd: VersionCommand,
+) -> miette::Result<()> {
+    let controller_route = &CloudOpts::route();
+    let node_name = start_embedded_node(ctx, &opts, None).await?;
+
+    // Lookup project
+    let id = match &opts.state.projects.get(&cmd.name) {
+        Ok(state) => state.config().id.clone(),
+        Err(_) => {
+            refresh_projects(ctx, &opts, &node_name, &CloudOpts::route(), None).await?;
+            opts.state.projects.get(&cmd.name)?.config().id.clone()
+        }
+    };
+
+    // Send request
+    let mut rpc = RpcBuilder::new(ctx, &opts, &node_name).build();
+    rpc.request(api::project::version(&id, controller_route))
+        .await?;
+    let project = rpc.parse_and_print_response::<Project>()?;
+    opts.state
+    .projects
+    .overwrite(&project.name, project.clone())?;
+    delete_embedded_node(&opts, rpc.node_name()).await;
```

Comment:
After this line, you should write the command output, something along these lines (example from the [`project delete`](https://github.com/build-trust/ockam/blob/e924084b8fde9a2ccf07fa5fe4753b8ec6707281/implementations/rust/ockam/ockam_command/src/project/delete.rs#L96) command:
```rust
      opts.terminal
            .stdout()
            .plain(fmt_ok!(
                "Project with name '{}' has been deleted.",
                &cmd.project_name
            ))
            .machine(&cmd.project_name)
            .json(serde_json::json!({ "project": { "name": &cmd.project_name } }))
            .write_line()?;
```